### PR TITLE
Render NextLink for internal links

### PR DESCRIPTION
**Pull request by gustaveen**
## Justify why they are needed
Faster navigation for internal routes with next/link
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2586** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2586" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2586?utm_source=stack-comment).

File: apps/store/src/blocks/RichTextBlock/RichTextBlock.tsx [hunk 1]
```diff
@@ -16,6 +17,30 @@ export const richTextRenderOptions: RenderOptions = {
...
+      // External links
+      if (href?.match(/^(https?:)?\/\//)) {
+        return (
+          <a href={href} target={target}>
+            {children}
+          </a>
+        )
+      }
+
+      // Internal links
```
Comment (robinandeer) [inline apps/store/src/blocks/RichTextBlock/RichTextBlock.tsx:36]: Can we fix the issue with trailing `/` (slash) that Peter mentioned now?

Should be possible to check if `is_startpage` or something and remove trailing slash from the link

**Review by robinandeer — APPROVED** [review]
Nice work, looks clean
Added via Giphy

File: apps/store/src/blocks/RichTextBlock/RichTextBlock.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/blocks/RichTextBlock/RichTextBlock.tsx:36]: Ah, good point 👍

**Pull request merged**